component/admin/views/icals/tmpl/edit.php

```diff
@@ -99,7 +99,7 @@ function submitbutton(pressbutton) {
 
         <div class="control-group">
             <div class="control-label">
-                <?php echo JText::_("Select_Default_Category"); ?>
+                <?php echo JText::_("JEV_FALLBACK_CATEGORY"); ?>
```

Comment:
I don't want us to use "fallback" - non-native english speakers will find it difficult.  I prefer "Default" and clarify the explanations if needed